src/pybamm/input/parameters/lithium_ion/Ai2020.py

```diff
@@ -670,6 +670,7 @@ def get_parameter_values():
         "Initial concentration in negative electrode [mol.m-3]": 24108.0,
         "Initial concentration in positive electrode [mol.m-3]": 21725.0,
         "Initial temperature [K]": 298.15,
+        "Cell heat capacity [J.K-1.m-3]": 2.5e6,
```

Comment:
In general, we try not to add parameters that were not in the original paper to the parameter sets, so could you delete this from all the parameter sets?